index.bs

```diff
@@ -384,6 +384,56 @@ scheme, is encoded in the <code>URL</code> field.
   };
 </pre>
 
+<div class="example">
+In this example we build a page supporting a device. To start with when the page
```

Comment:
There's a missing comma after "with", but it might read better as "When the page is first loaded, it should check ..."